tests/integrations/httpx/test_httpx.py

```diff
@@ -301,7 +300,8 @@ def test_do_not_propagate_outside_transaction(sentry_init, httpx_mock):
     httpx_client.get("http://example.com/")
 
     request_headers = httpx_mock.get_request().headers
-    assert "sentry-trace" not in request_headers
+    assert "sentry-trace" in request_headers
+    assert request_headers["sentry-trace"] == sentry_sdk.get_traceparent()
```

Comment:
Do you know why we were expecting to not propagate in the original version of this test case? The updated behavior is what I'd have expected to always be the case

Reply:
was part of this change
https://github.com/getsentry/sentry-python/pull/3070
and I believe this was an unintended side effect of that change...

but now we fixed the span without transaction problem another way with `only_if_parent` and this is the desired twp behaviour.